mobile/src/main/java/com/wedriveu/mobile/model/User.java

```diff
@@ -3,7 +3,9 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 /**
- * Created by Marco on 12/07/2017.
+ * <p>
+ *     Describes a logged user
```

Comment:
I think you wrote this class. I didn't know whether was a right move to edit it.
I'll write your name.